Intro_markdown.Rmd

```diff
@@ -213,31 +209,31 @@ To know more about a package, it is always useful to read the associated documen
 
 You can now see by looking in the environment window that an object has been created (the offenders dataset), and that it has 1413 observations and 9 variables.
 
-Now look at the Console tab. You should see the commands library and read_csv() appear with the whole path to the data set. It is a good idea to copy and paste these commands inside your script, so you won't need to do this again to load the data.
+Now look at the Console tab. You should see the commands library and `read_csv()` appear with the whole path to the data set. It is a good idea to copy and paste these commands inside your script, so you won't need to do this again to load the data.
 
-Alternatively you can simply use the function read_csv() from the readr package (which is included as part of tidyverse - see section 2.2):
+Alternatively you can simply use the function `read_csv()` from the `readr` package (which is included as part of tidyverse - see section 2.2:
 
 ```{r eval = FALSE, results="hide"}
 offenders <- readr::read_csv("Offenders_Chicago_Police_Dept_Main.csv")
 ```
 
 Note that the above assumes that the csv file is in your working directory, otherwise you will need to include the file path - see section 2.1.
 
-### Importing data from the Analytical Platform cloud storage (Amazon s3)
+### Importing data from the Analytical Platform cloud storage (Amazon S3)
 
-Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon s3, which is the cloud storage solution used by the Analytical Platform. To import the data for this session from the 'alpha-r-training' s3 bucket use this command:
+Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. To import the data for this session from the 'alpha-r-training' S3 bucket use this command:
 
 ```{r, results="hide"}
 offenders <- botor::s3_read("s3://alpha-r-training/intro-r-training/Offenders_Chicago_Police_Dept_Main.csv", read.csv)
 ```
 
-Both this and the code shown above are suitable for csv files, so it is assumed by default that the first line of the file contains a header (header = T) and the columns are separated by a comma symbol (sep = “,”). The latter approach assumes that you have already loaded the readr package, as it is part of tidyverse - see section 2.2.
+Both this and the code shown above are suitable for csv files, so it is assumed by default that the first line of the file contains a header (header = T) and the columns are separated by a comma symbol (sep = “,”). The former approach assumes that you have already loaded the `readr` package, as it is part of `tidyverse` - see section 2.2.
```

Comment:
Re the final sentence - not sure if this is needed anymore, since that fact that readr is required for the former approach is already mentioned in the subsection above.

On the other hand, the sentence could be adapted to refer to the code in this section by changing `read.csv` to `readr::read_csv` (assuming that doesn't break anything).

Then it'd be more consistent if we use only `readr::read_csv` rather than introducing `read.csv` as well.

Reply:
Good spot, have removed the redundant final sentence.

Good idea to make it consistent with `readr::read_csv`, but some columns get imported differently (no integer, numeric or character) so will have to update 2.5 as well (hopefully nothing else needs updating then too)

Reply:
Think i've updated everything that needs to be